.gitignore

```diff
@@ -2,3 +2,5 @@ config.yml
 yet-another-cloudwatch-exporter
 vendor
 dist
+yace
+.idea/
```

Comment:
Could you please add .idea to your global .gitignore. I think this is nothing project specific or am i missing something?

Reply:
Removed